.gitignore

```diff
@@ -14,3 +14,4 @@
 .rspec
 mkmf.log
 Guardfile
+.idea
```

Comment:
oooooh, do tell. What's a dot idea?

Reply:
I think this is for Christa's rubymine setup!

Reply:
Oooooh gotcha!

Reply:
Yep, that's correct! It's a common editor at CfA so figured I'd toss in